apps/studio/src/stories/Page/SignInPage/SingpassCallbackPage.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import { authSingpassHandlers } from "tests/msw/handlers/auth/singpass"
+import { meHandlers } from "tests/msw/handlers/me"
+
+import { withChromaticModes } from "@isomer/storybook-config"
+
+import SingpassCallbackPage from "~/pages/sign-in/singpass/callback"
+import { createSingpassEnabledGbParameters } from "~/stories/utils/growthbook"
+
+const meta: Meta<typeof SingpassCallbackPage> = {
+  title: "Pages/Sign In Page/Singpass Callback Page",
+  component: SingpassCallbackPage,
+  parameters: {
+    loginState: false,
+    growthbook: [createSingpassEnabledGbParameters(true)],
+    msw: {
+      handlers: [
+        meHandlers.unauthorized(),
+        authSingpassHandlers.callback.default(),
+      ],
+    },
+    nextjs: {
+      router: {
+        query: {
+          state: "state",
+          code: "code",
+        },
+      },
+    },
+  },
+}
+
+export default meta
+type Story = StoryObj<typeof SingpassCallbackPage>
+
+export const NewUser: Story = {
+  parameters: {
+    chromatic: withChromaticModes(["gsib", "mobile"]),
+  },
```

Comment:
maybe move it into `const meta: Meta<typeof SingpassCallbackPage>`

Reply:
d31f2e21cec1b59475be7c2903ead8357b041322